influxdb_iox/tests/end_to_end_cases/remote.rs

```diff
@@ -230,6 +202,27 @@ async fn remote_store_get_table() {
     .await
 }
 
+/// Asserts that the directory contains metadata and parquet files for
+/// 1 parquet files: 1 partition that has 1 table with 1 parquet files
```

Comment:
I feel like this comment has too many "parquet file" repetitions? And tables have partitions, not the other way around, right?

```suggestion
/// Asserts that the directory contains metadata and files for
/// 1 table that has 1 partition with 1 parquet file
```

Reply:
in c3553e348 👍